### PR TITLE
fix li_wand_2016 target_transforms

### PR DESCRIPTION
**Pull request by pmeier**
With `impl_params=True` we cannot use the default `pystiche.ops.MRFOperator.scale_and_rotate_transforms`, but need to use custom code instead. With this the replication now yields almost the same results as the reference implementation:

# Theirs

res_3_100

# Our

fig_6__top

Todo:

- [x] tests
- [x] documentation

### CODE REVIEW

File: replication/li_wand_2016/main.py [hunk 1]
```diff
@@ -29,14 +28,13 @@ def figure_6(args):
...
-        # https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L65-L70
+        # https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L67
         style_loss_kwargs = {
-            "target_transforms": MRFOperator.scale_and_rotate_transforms(
-                num_scale_steps=1,
-                scale_step_width=5e-2,
-                num_rotate_steps=1,
-                rotate_step_width=7.5,
-            )
+            "patch_size": 2,
```
Comment (jbueltemeier) [inline replication/li_wand_2016/main.py:33]: Should this not be `patch_size=3`? This should correspond to the 9 entry in line [L67](https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L67).

File: replication/li_wand_2016/main.py [hunk 2]
```diff
@@ -29,14 +28,13 @@ def figure_6(args):
...
+            "stride": 1,
```
Comment (jbueltemeier) [inline replication/li_wand_2016/main.py:34]: Same as above. This should be `stride=2`.

**Review by jbueltemeier — COMMENTED** [review]
You are right with this transformation the images look very similar, only the third eye on the chin is still missing. However, I have the assumption that my comments will eventually fix this.

File: replication/li_wand_2016/main.py (hunk 1 above)
Comment (pmeier) [reply replication/li_wand_2016/main.py:33]: Two things:

1. My comment referenced the wrong line. [L66]() is the relevant one.
2. Regardless of that, my local copy had these changes and I just used this. Good catch!

File: replication/li_wand_2016/main.py (hunk 2 above)
Comment (pmeier) [reply replication/li_wand_2016/main.py:34]: Same as above.

**Comment by pmeier** [comment]
With the corrected parameters we get the following results (theirs / ours)

res_3_100 fig_6__top

**Review by jbueltemeier — APPROVED** [review]

**Pull request merged**